### ISSUE TITLE
Update border color for user input elements

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/heracles-etd#242]
SODA reports that the border color for user input elements (`<textarea>`, file upload `<input>`, checkbox `<input>`, dropdown `<select>`) does not achieve 3:1 color contrast with the adjacent white background.

This border color is currently specified by ` --bs-border-color: #dee2e6;` which is also used elsewhere throughout the application (e.g., the border-bottom on a table row, which does not require contrast).

If possible, we should update the border color for _only_ the user input elements mentioned above. `--stanford-60-black: #767674` achieves sufficient contrast.

### Reported by SODA
> 
> [Summary]
> Active user interface component lacks 3 to 1 contrast ratio
> 
> [Description]
> "The visual boundary of an active user interface component lacks 3 to 1 contrast ratio.
> Checkboxes such as ""I have read and agree to the terms of the Stanford University license"" do not have enough contrast.  
> 
> Foreground: #DEE2E6
> Background: #FFFFFF
> Contrast ratio: 1.302:1"
> 
> [Impact]
> Serious
> 
> [Recommended to fix]
> "Rule : 
> The visual characteristics that indicate the component's purpose or type (e.g. is it a checkbox, text field, slider, switch, etc.) must have sufficient contrast of 3 to 1 with the adjacent background. Exceptions exist.
> 
> How To Fix: 
> Fix this issue by adjusting the user interface component boundary and/or background to increase the contrast with either the inner or outer adjacent background to at least 3 to 1.
> 
> Reference: 
> WCAG Understanding document: https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast
> 
> Background: 
> People who have low vision or are colorblind may have difficulty perceiving that an element is interactive or what its state is - e.g. whether it has keyboard focus, is selected/checked/pressed, etc. - if the contrast between the element boundaries and/or state indicators and its background or adjacent colors is insufficient. When an interactive element's visual boundaries and state indicators have adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive which elements on the screen are interactive and what their current state is."
